client/src/pages/ProfilePage/ProfilePage.js

```diff
@@ -1,17 +1,36 @@
 import React, { Component } from "react";
 import Header from "../../components/Header/Header.js";
 import Footer from "../../components/Footer/Footer.js";
+import { Link , BrowserRouter as Router} from "react-router-dom";
+import { Pp , ProfileImg , EditButton ,createButton } from "./ProfilePage.style.js";
+import photo from "../../assets/profilePhotos/HELLOMATE.jpg";
+import edit from "../../assets/images/editt.png";
+import createGroup from "../../assets/images/create.png";
 
 class ProfilePage extends Component {
+
   render() {
+    console.log(this.props.users[0]);
     return (
-      <div>
-        <Header />
-        <p> Make my profile!!</p>
-        <Footer />
-      </div>
+      <Router>
```

Comment:
also could you explain why the a Router element is here?

Reply:
well i've already removed it locally